en/identity-server/7.1.0/docs/apis/organization-apis/scim2/index.md

```diff
@@ -0,0 +1,7 @@
+# User management
+
+The SCIM2 Rest APIs of {{product_name}} implements the SCIM 2.0 protocol for organizations according to the [specification](https://datatracker.ietf.org/doc/html/rfc7644){target="_blank"}. This section provides information on the following APIs.
+
+- [SCIM2 organization users API]({{base_path}}/apis/organization-apis/scim2/scim2-org-user-mgt/)
+- [SCIM2 organization groups API]({{base_path}}/apis/organization-apis/scim2/scim2-org-group-mgt/)
+- [SCIM2 organization bulk API]({{base_path}}/apis/organization-apis/scim2/scim2-org-bulk-mgt/)
```

Comment:
Add new line

Reply:
fixed with https://github.com/wso2/docs-is/pull/5141/commits/b32df84ad27bbeef6037a183aebc2f68824d5422